web_widget_x2many_2d_matrix/static/src/js/web_widget_x2many_2d_matrix.js

```diff
@@ -50,10 +53,18 @@ openerp.web_widget_x2many_2d_matrix = function(instance)
             this.field_y_axis = node.attrs.field_y_axis || this.field_y_axis;
             this.field_label_x_axis = node.attrs.field_label_x_axis || this.field_x_axis;
             this.field_label_y_axis = node.attrs.field_label_y_axis || this.field_y_axis;
+            this.x_axis_clickable = node.attrs.x_axis_clickable != undefined ? this.parse_boolean(node.attrs.x_axis_clickable) : this.x_axis_clickable;
+            this.y_axis_clickable = node.attrs.y_axis_clickable != undefined ? this.parse_boolean(node.attrs.y_axis_clickable) : this.y_axis_clickable;
```

Comment:
No, it doesn't apply here. We only want to have a default in case you don't specify this attribute in the XML. If the widget user is putting specifically "0" in the option, then it should be passed to `parseBoolean` function, which is what is done right now.